lib/Driver/UnixToolChains.cpp

```diff
@@ -325,7 +325,10 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   Arguments.push_back(
       context.Args.MakeArgString(context.Output.getPrimaryOutputFilename()));
 
-  return {Clang, Arguments};
+  InvocationInfo II{Clang, Arguments};
+  II.allowsResponseFiles = true;
```

Comment:
Why true? Comment?

Reply:
If there are a bunch of similar lines, could you please factor them into an intention-revealing method? Or add a comment to each?

Reply:
Never mind, see below.